audit.py

```diff
@@ -862,13 +993,54 @@ def convert_alert(alert, verbose):
             "severity": alert.rule.severity,
         },
         "most_recent_instance": convert_alert_instance(alert.most_recent_instance),
+        "instances": [convert_alert_instance(instance) for instance in alert.get_instances()] if verbose else [],
     }
-    if verbose:
-        converted["instances"] = [convert_alert_instance(instance) for instance in alert.get_instances()]
     return converted
 
 
-def empty_alert():
+class AlertFilter:
+
+    def __init__(
+            self,
+            *,
+            state: State | None = None,
+            security_severity_level: SecuritySeverityLevel | None = None,
+            severity: Severity | None = None,
+    ):
```

Comment:
```suggestion
    ) -> None:
```

Reply:
__init__() by definition does not return anything, so why add a type hint here?